src/main/java/Task.java

```diff
@@ -0,0 +1,40 @@
+import java.time.LocalDate;
+
+public class Task {
+    protected String msg;
```

Comment:
Perhaps "message" is a more descriptive name for "msg". Since it is a class-wide variable, I think it should have a more descriptive name